oauth2client/service_account.py

```diff
@@ -145,14 +152,20 @@ def _to_json(self, strip, to_serialize=None):
             strip, to_serialize=to_serialize)
 
     @classmethod
-    def _from_parsed_json_keyfile(cls, keyfile_dict, scopes):
+    def _from_parsed_json_keyfile(cls, keyfile_dict, scopes,
+                                  token_uri=GOOGLE_TOKEN_URI,
```

Comment:
They're always passed; it seems more convenient to be able to change the default in fewer places, should that ever need to happen.  I don't feel strongly about it, though.